SPECS/pcaudiolib/pcaudiolib.spec

```diff
@@ -1,15 +1,13 @@
 Summary:        Portable C Audio Library
 Name:           pcaudiolib
-Version:        1.1
+Version:        1.2
 Release:        1%{?dist}
 License:        GPLv3
```

Comment:
Would you mind changing this to be "GPLv3+", to match the licensing given in the [`README`](https://github.com/espeak-ng/pcaudiolib)? It seems like I missed the "or later" bit when I originally wrote the spec.